Dockerfile

```diff
@@ -326,27 +320,43 @@ ARG PRODUCT
 ENV PRODUCT=${PRODUCT}
 ARG DEFAULT_PRODUCT_LICENSE
 ENV DEFAULT_PRODUCT_LICENSE=${DEFAULT_PRODUCT_LICENSE}
+ARG DOCKER_BUILDTAGS
+ENV DOCKER_BUILDTAGS="${DOCKER_BUILDTAGS}"
+ENV PREFIX=/build
+# TODO: This is here because hack/make.sh binary copies these extras binaries
+# from $PATH into the bundles dir.
+# It would be nice to handle this in a different way.
```

Comment:
Perhaps instead of copying these to `/usr/local/bin` we could just add the `bundles` directory to `PATH`, similar to what I did in https://github.com/docker/cli/pull/419 (not sure if there's code that expects the binaries to be in `/usr/local/bin` though)

Reply:
Here is the logic: https://github.com/moby/moby/blob/c36460c437c8c515c543dd31afcbb5c2a9f5dd48/hack/make/binary-daemon#L4-L31

So we can add them anywhere into PATH, or add anywhere to PATH, or just copy them manually.
I wanted to reduce redundant code as much as possible here.
The other thing that makes it *kinda* tricky is each build target builds in a subdir, like `bundles/binary-daemon` and `bundles/dynbinary-daemon`, so I'd have to do this copy twice.

I could bind-mount the files in and update PATH to include each one, but this would only reduce the amount of copying we are doing.

Reply:
Basically my worry here is we have this list of commands that need to be copied as part of a build and different lists get out of sync.